apps/fulfillment-e2e/src/integration/fully-picking-a-picklist.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+describe('Fully pick a picking list', () => {
+  const pickingListId = '37cb241e-f18a-5768-985c-a2d7aff4875e';
+  const pickingFragment = new PickingFragment();
+
+  beforeEach(() => {
+    cy.visit(`/picking-list/picking/${pickingListId}`);
+  });
+
+  describe('Picking all product items', () => {
+    beforeEach(() => {
+      [pickingFragment.getProducts()].forEach((product) => {
+        const oryxCartQuantity = product.find('oryx-cart-quantity-input');
+        const increaseButton = oryxCartQuantity.find('[part="increase"]');
```

Comment:
Should be moved in page object / page fragment (we don't want to have selectors in test files)